src/helpers/getVisibleTodos.tsx

```diff
@@ -0,0 +1,22 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+export
+function getVisibleTodos(todos: Todo[], currentFilter: string) {
```

Comment:
Consider using 'const' instead of 'function' to declare the function.

Reply:
Done.